docs/book/component-guide/deployers/kubernetes.md

```diff
@@ -0,0 +1,433 @@
+---
+description: Deploy ZenML pipeline services on Kubernetes clusters.
+---
+
+# Kubernetes Deployer
+
+The Kubernetes deployer is a [deployer](./) flavor provided by the Kubernetes
```

Comment:
```suggestion
The Kubernetes deployer is a [deployer](./README.md) flavor provided by the Kubernetes
```

I think (possibly am mistaken?) you can't just link to the dir for this link to work.